src/colorizer/utils/url_utils.ts

```diff
@@ -61,9 +61,7 @@ export enum UrlParam {
 const ALLEN_FILE_PREFIX = "/allen/";
 const ALLEN_PREFIX_TO_HTTPS: Record<string, string> = {
   // eslint-disable-next-line @typescript-eslint/naming-convention
-  "/allen/aics/assay-dev": "https://dev-aics-dtp-001.int.allencell.org/assay-dev",
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  "/allen/aics/microscopy": "https://dev-aics-dtp-001.int.allencell.org/microscopy",
+  "/allen/aics/": "https://dev-aics-dtp-001.int.allencell.org/",
```

Comment:
is it possible that windows users will put in unc paths with backslashes?    Also sometimes I have seen a doubled forward slash before allen.
`//allen/aics/`
`\\\\allen\\aics\\`

Reply:
UNC paths are handled by the method `normalizeFilePathSlashes`! I tried a couple different combinations and double leading slashes work just fine.

The only configuration that didn't work was the quadruple-leading-backslash:

![image](https://github.com/user-attachments/assets/f285bf14-5bd5-4708-ae1f-dfb6ec5684df)

Reply:
Fixed!